src/database/models.py

```diff
@@ -1,2613 +1,2917 @@
 from django.db import models
+from django.db.models.fields import related
 from database.utils.constants import *
 from .utils.common import json_loader, get_json_if_not_none, get_summary_info
 from django.forms.models import model_to_dict
 from carbon_calculator.models import Action as CCAction
 import uuid
 
-CHOICES = json_loader('./database/raw_data/other/databaseFieldChoices.json')
-ZIP_CODE_AND_STATES = json_loader('./database/raw_data/other/states.json')
+CHOICES = json_loader("./database/raw_data/other/databaseFieldChoices.json")
+ZIP_CODE_AND_STATES = json_loader("./database/raw_data/other/states.json")
 
 
 class Location(models.Model):
-  """
```

Comment:
i think u may have formatted this file with ur editor.  Can u undo?

Reply:
yes, that is what I explained earlier, and No, it is too far ahead to undo. Also, is there a reason we are choosing not to format that file? Can you come up with a default indentation that everyone can use, so that this isnt a problem anymore?

Reply:
Agree with Sam, makes it very hard to see what the changes are when the entire file changes.  
That said, if we decide to do a standard formatting (that we agree to), let's do that as a separate PR